Stylesheets/PageStyles/authentication.css

```diff
@@ -1,13 +1,64 @@
+
 .main {
-    padding-top: 10rem;
+    grid-template-columns: auto;
+   
+}
+
+.background {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    filter: blur(1px);
+}
+
+.product-gryffindor {
+    background-color: #800000;
+    width: 100%;
+    height: 50vh;
+}
+
+.product-slytherin {
+    background-color: #294936;
+    width: 100%;
+    height: 50vh;
+}
+
+.product-ravenclaw {
+    background-color: #015783;
+    width: 100%;
+    height: 50vh;
+}
+
+.product-hufflepuff {
+    background-color: #FFD63B;
+    width: 100%;
+    height: 50vh;
+}
+
+.product {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column
+}
+
+.background, .auth-cards {
+    grid-area: 1/1/2/2;
 }
 
 .auth-cards {
+    margin: auto;
+    display: flex;
     flex-direction: column;
     align-items: center;
+    justify-content: center;
```

Comment:
This can also be made as separate utility class

Reply:
Yes changing this.